plugins/cicd-statistics/src/charts/status-chart.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Fragment, useMemo } from 'react';
+import {
+  Bar,
+  ComposedChart,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Legend,
+  LegendProps,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+import Alert from '@material-ui/lab/Alert';
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+} from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { formatISO9075, parseISO } from 'date-fns';
```

Comment:
I recall there was discussion on `date-fns` when date libraries were being decided upon, and the result was this ADRs:

https://backstage.io/docs/architecture-decisions/adrs-adr010
https://backstage.io/docs/architecture-decisions/adrs-adr012

Not suggesting to prevent a merge, per se, but this might need to be refactored later if it can't be moved to Luxon now?

Reply:
I've missed those ADR's, this makes total sense! I'll change to using Luxon and standardised formatting.